lib/core/presentation/pages/create_post/create_post_page.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:app/gen/assets.gen.dart';
+import 'package:app/theme/color.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:auto_route/annotations.dart';
+import 'package:flutter/material.dart';
+
+import '../../../../i18n/i18n.g.dart';
+
+@RoutePage()
+class CreatePostPage extends StatelessWidget {
+  const CreatePostPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final textCtrl = TextEditingController();
+    final t = Translations.of(context);
+    return Scaffold(
+      backgroundColor: LemonColor.black,
+      appBar: AppBar(
```

Comment:
Please check on using LemonAppBar, already designed-like back button

Reply:
LemonAppBar are using specified space for icons only. We having button here so cannot re-use it

Reply:
As I remember it allows to pass actions as a list of widgets  :/